src/frontend/src/tables/part/PartTable.tsx

```diff
@@ -349,6 +357,19 @@ export function PartListTable({
 
   const [selectedPart, setSelectedPart] = useState<number>(-1);
 
+  const {
```

Comment:
I think you do not need to separately fetch the part instance - it's already available in the "record" instance when the action is triggered.